android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrTypography.kt

```diff
@@ -76,12 +76,12 @@ val SubletrTypography = Typography(
 		color = Color.Gray,
 		fontFamily = FontFamily.Default,
 		fontWeight = FontWeight.Medium,
-		fontSize = 11.sp,
+		fontSize = 15.sp,
```

Comment:
Are you sure this isn't used anywhere else?
If it is used somewhere else, it might mess up another screen.

If you need a new text style, just make a new variable instead.

Reply:
I'm sure this wasn't used anywhere else BUT I agree another typography style is better